tests/closures-as-members-repeated-reallocation.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+Testing closure members reallocation
+--DESCRIPTION--
+This test verifies that closures can be set as members repeatedly
+--FILE--
+<?php
+class TestClosure extends \Threaded implements \Collectable {
```

Comment:
nit: we don't have a Collectable interface anymore

Reply:
my fault, should have checked that